init.lua

```diff
@@ -0,0 +1,397 @@
+local lazypath = vim.fn.stdpath("data") .. "/lazy/lazy.nvim"
```

Comment:
I apologize if this is a silly question, but how will you handle the faith/vim-go package with this structure?

Reply:
I replaced most of vim-go functionality with lsp, treesitter. I might extract some functionality from vim-go into nvim specific plugins, but didn't decided yet.